packages/headless/src/features/commerce/search/search-actions-thunk-processor.ts

```diff
@@ -126,15 +126,15 @@ export class AsyncSearchThunkProcessor<RejectionType> {
 
```

Comment:
These changes are only to accomodate the fact that `originalQuery` and `correctedQuery` can actually be null.